src/components/SlateEditor/plugins/concept/SlateConceptPreview.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { FC, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from '@emotion/styled';
+import { spacing } from '@ndla/core';
+import { DeleteForever } from '@ndla/icons/editor';
+import { Link as LinkIcon } from '@ndla/icons/common';
+import { injectT, tType } from '@ndla/i18n';
+import {
+  NotionDialogContent,
+  NotionDialogText,
+  NotionDialogLicenses,
+} from '@ndla/notion';
+import Tooltip from '@ndla/tooltip';
+import { addShowConceptDefinitionClickListeners } from '@ndla/article-scripts';
+import { Concept as ConceptType } from '../../editorTypes';
+import IconButton from '../../../IconButton';
+
+const StyledFigureButtons = styled('div')`
+  position: absolute;
+  top: 0;
+  z-index: 1;
+  right: -${spacing.spacingUnit * 1.5}px;
+  margin-right: 40px;
+  margin-top: ${spacing.xsmall};
+
+  > * {
+    margin-bottom: ${spacing.xsmall};
+  }
+`;
+
+interface Props {
+  concept: ConceptType;
+  handleRemove: () => void;
+  id: number;
+}
+
+const SlateConceptPreview: FC<Props & tType> = ({
+  concept,
+  handleRemove,
+  id,
+  t,
+}) => {
+  useEffect(() => {
+    addShowConceptDefinitionClickListeners();
+  }, []);
+
+  return (
+    <>
```

Comment:
Har ikke tatt med visuelt element i denne, burde den være med?

Reply:
Kanskje. Er det nok å inludere VisualElement-komponenten til Ingeborg?

Reply:
Det virker med bilde men ser ut som om det er litt mer som skal til for Video? Prøvde å se raskt på `getVisualElement` i `PreviewConceptLightbox` i tillegg til komponenten, er det kanskje det som må til? @ingeborgsteel 

I såfall er jeg for det å flytte det ut i en egen komponent om det var det du mente tidligere i din pr Ingeborg? :/

Reply:
har testet i denne, den nederste har en video 
https://editorial-frontend-pr-841.ndla.sh/subject-matter/learning-resource/22665/edit/nb
http://localhost:3000/concept/717/edit/nb

Reply:
Noen som hadde noe formening/ erfaring med dette? :-) @ingeborgsteel @gunnarvelle ellers tenkte jeg se på det nå.

Mener det var relatert til dette trellokortet: https://trello.com/c/6Zm2a2fv/779-forklaringer-i-ed-fungerer-litt-rart

Reply:
Eg spurte på kortet, men har ikkje fått svar. Tenker vi kan gå videre slik det er no og heller endre seinere om dei ønsker det.

Reply:
oki, takk for svar 👍

Reply:
Så ikke at du hadde tagga meg her, men fant du ut av det?:')